open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -39,7 +39,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #36.        if len(df[df['Embarked'] == 'Z']) &gt; 0:
</pre><p>Please format the code by following this https://github.com/open-academy/machine-learning/blob/main/STYLE_GUIDE.md#code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/161/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>